R/ark.R

```diff
@@ -36,9 +40,11 @@
 #' }
 ark <- function(db_con, dir, lines = 10000L, 
                 compress = c("bzip2", "gzip", "xz", "none"),
-                tables = list_tables(db_con)){
+                tables = list_tables(db_con),
+                use_alternate = FALSE){
```

Comment:
Perhaps a `method = c("keep-open", "window", "sql-window", "manual-window")` would give more flexibility in the long run? I'm not in love with `"keep-open"`, but let's think of a better name for the method than `kirill` ;-)

Reply:
nice, I like this.  I struggled with concise meaningful names and clearly didn't come up with something.  `keep-open` is intuitive.  

I think you're suggesting here that "window" would be the method that uses `OFFSET`, and "sql-window" would be the method that uses `BETWEEN`?  (might be good to bipass my 'automatic' `has_between()` method.